test/e2e/e2e-test/docker/kafka/Dockerfile.provider

```diff
@@ -25,6 +25,8 @@ ADD ${DIST_PACKAGE} .
 RUN tar -zxf apache-skywalking-apm*.gz --strip=1 || true
 RUN mv apache-skywalking-apm-bin/* . || true
 
+RUN sed -i 's/# plugin.kafka.bootstrap_servers/plugin.kafka.bootstrap_servers/g' agent/config/agent.config
```

Comment:
Is this way good for you? `Check the agent.config` in the e2e process.

Reply:
Suggestion:
```bash
sed -i 's/#\s*plugin.kafka.bootstrap_servers/plugin.kafka.bootstrap_servers/g' agent/config/agent.config
```